lib/puppet.rb

```diff
@@ -211,7 +211,7 @@ def self.initialize_facts
   # code was deprecated in 2008, but this is still in heavy use.  I suppose
   # this can count as a soft deprecation for the next dev. --daniel 2011-04-12
   def self.newtype(name, options = {}, &block)
-    Puppet.deprecation_warning("Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead.")
+    Puppet.deprecation_warning(_("Creating #{name} via Puppet.newtype is deprecated and will be removed in a future release. Use Puppet::Type.newtype instead."))
```

Comment:
I am very much hoping they can be asked to ignore those parts. If not, we have an loootttt more work ahead of us. The interpolation functions correctly if the `#{...}` is left alone: Ruby performs the interpolation after it loads the translated string.